### PR TITLE
Add more info on geolocation resolution in docs

### PR DESCRIPTION
**Pull request by ekoyle**
Tell how location is resolved for geoloc foo.

### CODE REVIEW

File: doc/Extensions/Globe-Frontpage.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google";//Only one available at present
 ```
 
+Location resolution happens as follows (iff `$config['geoloc']['latlng'] == true;`):
```
Comment (murrant) [inline doc/Extensions/Globe-Frontpage.md]: iff typo here

Comment (ekoyle) [inline doc/Extensions/Globe-Frontpage.md]: It is short for "if and only if",  but I can drop the last 'f' if you'd
prefer.

Comment (murrant) [inline doc/Extensions/Globe-Frontpage.md]: Oh, I was not aware of that shortand.  Perhaps we should be more verbose for the documentation.

Comment (paulgear) [inline doc/Extensions/Globe-Frontpage.md]: +1 for writing it out in full.

**Comment by ekoyle** [comment]
I don't think there was enough value to increase the wordiness here, so I changed it to 'when'.

@Rosiak: Does this PR look accurate to you?

**Comment by murrant** [comment]
:+1:

**Pull request merged**